clang/lib/CodeGen/CodeGenTypes.h

```diff
@@ -106,9 +101,9 @@ class CodeGenTypes {
   }
   CodeGenModule &getCGM() const { return CGM; }
   ASTContext &getContext() const { return Context; }
-  const ABIInfo &getABIInfo() const { return TheABIInfo; }
+  const ABIInfo &getABIInfo() const;
```

Comment:
Given almost all the callers of getABIInfo() are calling `CGM.getTypes().getABIInfo()`, we probably want to just move this helper to CodeGenModule.

Reply:
Good point. I've addressed that. Thank you.